client/src/components/CertificateEarned/index.tsx

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import { createSelector } from 'reselect';
+import { connect } from 'react-redux';
+import { SuperBlocks } from '../../../../config/certification-settings';
+import {
+  isSignedInSelector,
+  userSelector,
+  userFetchStateSelector
+} from '../../redux/selectors';
+import CircleCheckRegular from './components/circle-check-regular';
+
+interface FetchState {
+  pending: boolean;
+  complete: boolean;
+  errored: boolean;
+}
+
+interface User {
+  isRespWebDesignCert: boolean;
+  is2018DataVisCert: boolean;
+  isApisMicroservicesCert: boolean;
+  isBackEndCert: boolean;
+  isDataAnalysisPyCertV7: boolean;
+  isDataVisCert: boolean;
+  isFrontEndCert: boolean;
+  isFrontEndLibsCert: boolean;
+  isFullStackCert: boolean;
+  isInfosecCertV7: boolean;
+  isInfosecQaCert: boolean;
+  isJsAlgoDataStructCert: boolean;
+  isMachineLearningPyCertV7: boolean;
+  isQaCertV7: boolean;
+  isRelationalDatabaseCertV8: boolean;
+  isSciCompPyCertV7: boolean;
+}
+
+const mapStateToProps = createSelector(
+  userFetchStateSelector,
+  isSignedInSelector,
+  userSelector,
+  (fetchState: FetchState, isSignedIn: boolean, user: User) => ({
+    fetchState,
+    isSignedIn,
+    user
+  })
+);
+
+const certsMap = {
+  [SuperBlocks.RespWebDesignNew]: false,
+  [SuperBlocks.RespWebDesign]: false,
+  [SuperBlocks.JsAlgoDataStruct]: false,
+  [SuperBlocks.JsAlgoDataStructNew]: false,
+  [SuperBlocks.FrontEndDevLibs]: false,
+  [SuperBlocks.DataVis]: false,
+  [SuperBlocks.BackEndDevApis]: false,
+  [SuperBlocks.RelationalDb]: false,
+  [SuperBlocks.QualityAssurance]: false,
+  [SuperBlocks.SciCompPy]: false,
+  [SuperBlocks.DataAnalysisPy]: false,
+  [SuperBlocks.InfoSec]: false,
+  [SuperBlocks.MachineLearningPy]: false,
+  [SuperBlocks.CodingInterviewPrep]: false
+};
+
+interface CertificateEarnedProps {
+  superBlock: SuperBlocks;
+  user: User;
+}
+
+const CertificateEarned = (props: CertificateEarnedProps): JSX.Element => {
+  const { superBlock, user } = props;
```

Comment:
maybe the type is lost, but it still doesn't make sense, does changing it to this help with the type error?

```suggestion
const CertificateEarned = ({superBlock, user}: CertificateEarnedProps) => {
```

Reply:
I tried to merge this but still fails the test. This is what is causing the linting to fail:
![image](https://user-images.githubusercontent.com/8081121/218233657-0757110f-0867-41b3-9b01-d1d8f0504536.png)

Reply:
I don't think that the `SuperBlocks.TheOdinProject` still exists.

Reply:
Will updating this PR branch with `main` resolve this issue? 🤔

Reply:
Let's test that, if it doesn't work, we can try turning `certsMap` into a Map.